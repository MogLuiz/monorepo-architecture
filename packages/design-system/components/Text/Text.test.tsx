import { customRender } from "@monorepo/test-commons/react-testing-library";
import { Text } from "./index";

const render = customRender();

describe("<Text />", () => {
  it("ensure render Text correctly", () => {
    const { container } = render(<Text tag="h1">Sample text</Text>);
    // screen.debug();
    expect(container).toMatchSnapshot();
  });
});
