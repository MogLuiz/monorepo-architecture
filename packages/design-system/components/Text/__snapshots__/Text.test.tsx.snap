// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Text /> ensure render Text correctly 1`] = `
.c0 {
  font-family: sans-serif;
}

<div>
  <h1
    class="c0"
  >
    Sample text
  </h1>
</div>
`;
